edx_rest_framework_extensions/tests/test_authentication.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+""" Tests for authentication classes. """
+
+import json
+
+import httpretty
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.test import override_settings, RequestFactory, TestCase
+from rest_framework.exceptions import AuthenticationFailed
+
+from edx_rest_framework_extensions.authentication import BearerAuthentication
+from edx_rest_framework_extensions.tests import factories
+
+OAUTH2_PROVIDER_URL = 'http://example.com/oauth2'
+
+User = get_user_model()
+
+
+class AccessTokenMixin(object):
+    """ Test mixin for dealing with OAuth2 access tokens. """
+    DEFAULT_TOKEN = 'abc123'
+
+    def mock_access_token_response(self, status=200, token=DEFAULT_TOKEN, username='fake-user'):
+        """ Mock the access token endpoint response of the OAuth2 provider. """
+        url = '{root}/access_token/{token}/'.format(root=OAUTH2_PROVIDER_URL, token=token)
```

Comment:
Here, the provider URL isn't strip()ed or rstrip()ed at all.  Why the discrepancy?